apps/website/src/components/courses/GroupSwitchModal.stories.tsx

```diff
@@ -43,7 +43,7 @@ const mockAvailableGroupsAndDiscussions: DiscussionsAvailable = {
       }),
       userIsParticipant: true,
       spotsLeftIfKnown: 0,
-      allDiscussionsHaveStarted: false,
+      isTooLateToSwitchTo: false,
```

Comment:
Unrelated: I renamed `hasStarted` and `allDiscussionsHaveStarted` to `isTooLateToSwitchTo` for clarity